optimum/habana/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -75,6 +75,8 @@ def prune_heads(self, heads):
         self.pruned_heads = self.pruned_heads.union(heads)
 
     def _attn(self, query, key, value, attention_mask=None, head_mask=None):
+        key = key.contiguous()
```

Comment:
with line 212, do we still need contiguous here?

Reply:
Yes, it still has perf drop if we remove this.